src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
I'm fine leaving this discussion to later; fixing the BootRequirementChecker (in bootloader module) is in the backlog, we can discuss there which one to use as well as individual checks.